packages/venia-ui/lib/components/RadioGroup/radio.js

```diff
@@ -13,24 +14,30 @@ export class RadioOption extends Component {
         classes: shape({
             input: string,
             label: string,
-            root: string
+            root: string,
+            icon: string
         }),
         label: node.isRequired,
         value: node.isRequired
     };
 
     render() {
         const { props } = this;
-        const { classes, id, label, value, ...rest } = props;
+        const { classes, id, initialChecked, label, value, ...rest } = props;
 
         return (
             <label className={classes.root} htmlFor={id}>
                 <Radio
                     {...rest}
                     className={classes.input}
                     id={id}
+                    defaultChecked={initialChecked}
+                    type="radio"
```

Comment:
Was this prop needed?

https://github.com/joepuzzo/informed/blob/master/src/components/form-fields/Radio.js#L49

Reply:
It's redundant, good call